src/status_im/contexts/onboarding/intro/view.cljs

```diff
@@ -19,19 +20,16 @@
                             (reset! scan-sync-code/dismiss-animations reset-top-animation-fn))
      :animations-duration constants/onboarding-modal-animation-duration
      :animations-delay    constants/onboarding-modal-animation-delay
-     :top-card            {:on-press            #(debounce/throttle-and-dispatch
-                                                  [:open-modal
-                                                   :screen/onboarding.sign-in-intro]
-                                                  2000)
+     :top-card            {:on-press            #(rf/dispatch [:open-modal
+                                                               :screen/onboarding.sign-in-intro])
                            :heading             (i18n/label :t/sign-in)
                            :animated-heading    (i18n/label :t/sign-in-by-syncing)
                            :accessibility-label :already-use-status-button}
      :bottom-card         {:on-press            (fn []
                                                   (when-let [blur-show-fn @overlay/blur-show-fn-atom]
                                                     (blur-show-fn))
-                                                  (debounce/throttle-and-dispatch
-                                                   [:open-modal :screen/onboarding.new-to-status]
```

Comment:
we were showing blur-overlay, but because of throttle we were not able to open the screen

Reply:
As now we already throttle all on-presses, we don't need debounce anymore.

(qa team, please make sure https://github.com/status-im/status-mobile/issues/18369 is still fixed)